src/Compilers/Core/Portable/InternalUtilities/Debug.cs

```diff
@@ -13,15 +14,21 @@ internal static class RoslynDebug
     {
         /// <inheritdoc cref="Debug.Assert(bool)"/>
         [Conditional("DEBUG")]
-        public static void Assert([DoesNotReturnIf(false)]bool b) => Debug.Assert(b);
+        public static void Assert([DoesNotReturnIf(false)] bool b) => Debug.Assert(b);
 
         /// <inheritdoc cref="Debug.Assert(bool, string)"/>
         [Conditional("DEBUG")]
-        public static void Assert([DoesNotReturnIf(false)]bool b, string message)
+        public static void Assert([DoesNotReturnIf(false)] bool b, string message)
             => Debug.Assert(b, message);
 
         [Conditional("DEBUG")]
-        public static void AssertNotNull<T>([NotNull]T value) where T : class?
-            => Debug.Assert(value is object, "Unexpected null reference");
+        public static void AssertNotNull<T>([NotNull] T value) where T : class?
```

Comment:
This is not really something to fix in this PR, but I don't understand why this method is generic.

Reply:
I suspect it is so that it cannot be called with a struct parameter. 
If it just took an `object` parameter, then structs would box, rather than produce an error.